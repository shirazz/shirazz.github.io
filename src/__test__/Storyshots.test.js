import initStoryshots from '@storybook/addon-storyshots';

// Avoid errors when using gatsby-link in a component.
global.___loader = {
  enqueue: jest.fn(),
};

initStoryshots();
