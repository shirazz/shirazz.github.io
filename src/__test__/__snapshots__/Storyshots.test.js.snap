// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Header default 1`] = `
<div
  className="header"
>
  <div
    className="inner"
  >
    <h1
      className="title is-2 has-text-weight-bold"
    >
      <a
        className="link"
        href="/"
        onClick={[Function]}
      >
        Gatsby
      </a>
    </h1>
  </div>
</div>
`;
